WebDriverAgentLib/Categories/XCUIElement+FBIsVisible.m

```diff
@@ -24,7 +24,7 @@ @implementation XCUIElement (FBIsVisible)
 
 - (BOOL)fb_isVisible
```

Comment:
Same as in `fb_isAccessibilityElement`, I think we should add a fallback to `fb_uniqueSnapshot` and `fb_lastSnapshot` here as well.